src/Enterspeed.Source.UmbracoCms/Services/UmbracoRichTextParser.cs

```diff
@@ -46,6 +47,12 @@ public string PrefixRelativeImagesWithDomain(string html, string mediaDomain)
                 var src = imageNode.GetAttributeValue("src", string.Empty);
                 if (src.StartsWith("/media/"))
                 {
+                    var absolutePath = mediaDomainUrl.AbsolutePath;
+                    if (!absolutePath.Equals("/"))
```

Comment:
If no relative path, we should only have "/" in the variable. All other results should indicate an actualy relative path